Xiaomi/RedmiNote9S/res/values/config.xml

```diff
@@ -0,0 +1,957 @@
+        <item>266.87</item>
+        <item>268.51</item>
+        <item>270.15</item>
+        <item>271.79</item>
+        <item>273.44</item>
+        <item>275.08</item>
+        <item>276.72</item>
+        <item>278.36</item>
+        <item>280</item>
+        <item>281.65</item>
+        <item>283.29</item>
+        <item>284.93</item>
+        <item>286.57</item>
+        <item>288.22</item>
+        <item>289.86</item>
+        <item>291.5</item>
+        <item>293.14</item>
+        <item>294.79</item>
+        <item>296.43</item>
+        <item>298.07</item>
+        <item>299.71</item>
+        <item>301.35</item>
+        <item>303</item>
+        <item>304.64</item>
+        <item>306.28</item>
+        <item>307.92</item>
+        <item>309.57</item>
+        <item>311.21</item>
+        <item>312.85</item>
+        <item>314.49</item>
+        <item>316.54</item>
+        <item>318.19</item>
+        <item>319.83</item>
+        <item>321.47</item>
+        <item>323.11</item>
+        <item>324.76</item>
+        <item>326.4</item>
+        <item>328.04</item>
+        <item>329.68</item>
+        <item>331.32</item>
+        <item>332.97</item>
+        <item>334.61</item>
+        <item>336.25</item>
+        <item>337.89</item>
+        <item>339.54</item>
+        <item>341.18</item>
+        <item>342.82</item>
+        <item>344.46</item>
+        <item>346.1</item>
+        <item>347.75</item>
+        <item>349.39</item>
+        <item>351.03</item>
+        <item>352.67</item>
+        <item>354.32</item>
+        <item>355.96</item>
+        <item>357.6</item>
+        <item>359.24</item>
+        <item>360.88</item>
+        <item>362.53</item>
+        <item>364.17</item>
+        <item>365.81</item>
+        <item>367.45</item>
+        <item>369.1</item>
+        <item>370.74</item>
+        <item>372.38</item>
+        <item>374.02</item>
+        <item>375.66</item>
+        <item>377.31</item>
+        <item>378.95</item>
+        <item>380.59</item>
+        <item>382.23</item>
+        <item>383.88</item>
+        <item>385.52</item>
+        <item>387.16</item>
+        <item>388.8</item>
+        <item>390.44</item>
+        <item>392.09</item>
+        <item>393.73</item>
+        <item>395.37</item>
+        <item>397.01</item>
+        <item>398.66</item>
+        <item>400.3</item>
+        <item>401.94</item>
+        <item>403.58</item>
+        <item>405.22</item>
+        <item>406.87</item>
+        <item>408.51</item>
+        <item>410.15</item>
+        <item>411.79</item>
+        <item>413.44</item>
+        <item>415.08</item>
+        <item>416.72</item>
+        <item>418.36</item>
+        <item>420</item>
+    </array>
+
+    <!-- Screen brightness used to dim the screen when the user activity
+         timeout expires.  May be less than the minimum allowed brightness setting
+         that can be set by the user. -->
+    <integer name="config_screenBrightnessDim">6</integer>
```

Comment:
6 looks pretty high to me, check what's in stock rom? You'll probably need to do /4 compared to stock rom because stock is likely in 10bits, we're 8bits